Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboardingV2.hs

```diff
@@ -121,11 +122,10 @@ getOnboardingConfigs (mbPersonId, _, merchantOpCityId) mbOnlyVehicle = do
   bikeConfigsRaw <- CQDVC.findByMerchantOpCityIdAndCategory merchantOpCityId DTV.MOTORCYCLE
   ambulanceConfigsRaw <- CQDVC.findByMerchantOpCityIdAndCategory merchantOpCityId DTV.AMBULANCE
 
-  cabConfigs <- mapM (mkDocumentVerificationConfigAPIEntity personLangauge) (filterVehicleDocuments cabConfigsRaw)
-  autoConfigs <- mapM (mkDocumentVerificationConfigAPIEntity personLangauge) (filterVehicleDocuments autoConfigsRaw)
-  bikeConfigs <- mapM (mkDocumentVerificationConfigAPIEntity personLangauge) (filterVehicleDocuments bikeConfigsRaw)
-  ambulanceConfigs <- mapM (mkDocumentVerificationConfigAPIEntity personLangauge) (filterVehicleDocuments ambulanceConfigsRaw)
-
+  cabConfigs <- filter (\doc -> (doc.validateUsing /= Just DTO.FRONTEND) || fromMaybe False makeSelfieAadhaarPanMandatory || doc.documentType /= DTO.ProfilePhoto && doc.documentType /= DTO.AadhaarCard && doc.documentType /= DTO.PanCard) <$> mapM (mkDocumentVerificationConfigAPIEntity personLangauge) (filterVehicleDocuments cabConfigsRaw)
```

Comment:
Can you move this filters as a Common function and reuse it for filtering cab, autos, bike and ambulance configs

Reply:
doc.documentType /= DTO.ProfilePhoto && doc.documentType /= DTO.AadhaarCard && doc.documentType /= DTO.PanCard

`not (documentType `elem` [DTO.ProfilePhoto, DTO.AadhaarCard, DTO.PanCard])`

Can you change it to this

Reply:
using notElem instead